packages/calling-component-bindings/src/hooks/usePropsFor.ts

```diff
@@ -61,7 +61,7 @@ import { ErrorBarSelector, errorBarSelector } from '../errorBarSelector';
 export const usePropsFor = <Component extends (props: any) => JSX.Element>(
```

Comment:
Are we saying that we don't support `usePropsFor` for components that need any Teams specific handlers, for now?

What is the plan for when we do have components that need Teams specific handlers (my guess would be to have a separate `usePropsForTeamsComponent` or some such, but want to confirm)?

Reply:
Yep, if there is anything teams specific, that would be usePropsForTeamsComponent(or some other names), but looks like calling sdk doesn't have any plan shipping something like this for now